app.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const path = require('path')
+const app = express()
+const PORT = process.env.PORT || 5000
+
+app.use(express.static('.'))
```

Comment:
Remember when you use a static server and use ```express.static('.')``` this allows the user to navigate through your file system that the server is running on. A better practice would be to use ```express.static('public')``` and move your main files into the public folder.